### PR TITLE
[export] Replicate exportdb examples and tests in oss.

### PR DESCRIPTION
**Pull request by zhxchen17**
Summary: Initial work to copy source to OSS for exportdb and make sure tests can run properly.

Test Plan: test_export

Differential Revision: D46369152

cc @voznesenskym @penguinwu @anijain2305 @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @Xia-Weiwen @wenzhe-nrv @jiayisunx @ipiszy

### CODE REVIEW

File: test/export/test_db.py [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+    @parametrize(
+        "name,case",
+        filter_examples_by_support_level(SupportLevel.SUPPORTED).items(),
+        name_fn=lambda name, case: "case_{}".format(name),
+    )
+    def test_exportdb_supported(self, name: str, case: ExportCase) -> None:
+        model = case.model
+        with torchdynamo.config.patch(asdict(_DynamoConfig())):
+            inputs = normalize_inputs(case.example_inputs)
+            exported_model, _ = torchdynamo.export(
```
Comment (angelayi) [inline test/export/test_db.py:44]: should this be torch.export eventually?

File: torch/_export/db/gen_example.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import os
```
Comment (angelayi) [inline torch/_export/db/gen_example.py:1]: what is this file for again?

File: test/export/test_db.py (hunk 1 above)
Comment (zhxchen17) [reply test/export/test_db.py:44]: yeah, I'm looking at @tugsbayasgalan

File: torch/_export/db/gen_example.py (hunk 2 above)
Comment (zhxchen17) [reply torch/_export/db/gen_example.py:1]: this is a script to generate a new example file with empty code.

**Review by angelayi — APPROVED** [review]
I think the tests are failing bc you need an empty __init__.py file in the export/db directory but everything else looks fine to me!

**Pull request closed without merging**